EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -37,26 +36,44 @@ struct MediaPlayerView: View {
     }
 
     @ViewBuilder private func renderActionButtons() -> some View {
-        HStack {
-            renderRewind()
-                .frame(maxWidth: .infinity, alignment: .leading)
-            if viewModel.playerState == .finished {
-                renderReplay()
-            } else {
-                renderPlayPause()
+        ZStack {
+            VStack {
+                HStack {
+                    // TODO: implment ActionButtons top row
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                HStack {
+                    renderRewind()
+                    Spacer()
+                    renderPlayPauseReplay()
+                    Spacer()
+                    renderForward()
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity)
+                HStack {
+                    renderSettings()
+                        .padding(.bottom, Constants.settingsIconPadding)
+                }
+                .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .bottomTrailing)
```

Comment:
Seems weird, we really should break into those smaller functions to make sense out of this